packages/react/src/CTAForm/CTAForm.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import {ComponentMeta, ComponentStory} from '@storybook/react'
+
+import {CTAForm} from './CTAForm'
+
+export default {
+  title: 'Components/CTAForm',
+  component: CTAForm,
+  args: {
+    inputLabel: 'Your work email address',
+    inputType: 'text',
+    placeholder: 'me@github.com',
+    buttonLabel: 'Subscribe',
+    checkboxLabel: (
+      <>
+        I agree to the <a href="www.github.com">terms and conditions</a> and <a href="www.github.com">privacy policy</a>
```

Comment:
Can we use InlineLink here, even though it's an example? Link colors are off here:

![Screenshot 2023-01-23 at 15 00 50](https://user-images.githubusercontent.com/13340707/214073188-2ee725cf-49aa-42e4-b743-0e250249332e.png)

Reply:
InlineLink might have a bug because when the screen size is full size then the text grows larger than its surrounding text. It goes to 20px after being 16px on smaller screen sizes. I can't figure out quite why yet.

Reply:
![image](https://user-images.githubusercontent.com/31220082/214083333-c5603ece-96aa-4a33-b7a7-c670f2ded589.png)
![image](https://user-images.githubusercontent.com/31220082/214083373-6a06c99d-e471-44e7-94c0-d1fc155092f4.png)